plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -761,6 +766,10 @@ public String toString() {
             }
             diskBuilder.append("/>\n");
 
+            if (_serial != null && !_serial.isEmpty() && _deviceType != DeviceType.LUN) {
```

Comment:
@wido see, we use the device type here to avoid adding the disk/volume serial if the device type is LUN